### PR TITLE
Added entity reset switch to toybox

### PR DESCRIPTION
**Pull request by ericrius1**
This switch resides on the wall behind the plant, and can be grabbed with hydra or clicked with mouse. When toggles, it will reset the scene, with this change, if one wishes to reset the toybox by just running a script, one should run https://raw.githubusercontent.com/highfidelity/hifi/master/unpublishedScripts/immediateClientReset.js as opposed to https://raw.githubusercontent.com/highfidelity/hifi/master/unpublishedScripts/masterReset.js

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4ab9c9932aacb7e8778b8db9aa7d0666dd49ef4a available at: https://gist.github.com/64f9a92e88bedc01e770<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6015.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6015.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: a07817e73676ef06c80595505647ec08d7051de2 available at: https://gist.github.com/64f9a92e88bedc01e770<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6015.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6015.dmg

File: libraries/script-engine/src/ScriptEngine.cpp [hunk 1]
```diff
@@ -974,9 +974,6 @@ void ScriptEngine::entityScriptContentAvailable(const EntityItemID& entityID, co
     QScriptValue entityScriptObject = entityScriptConstructor.construct();
     EntityScriptDetails newDetails = { scriptOrURL, entityScriptObject };
     _entityScripts[entityID] = newDetails;
-    if (isURL) {
```
Comment (ZappoMan) [inline libraries/script-engine/src/ScriptEngine.cpp]: duuuuuuuude! you can't do this!

File: libraries/fbx/src/FBXReader.h [hunk 2]
```diff
@@ -1,431 +1,431 @@
-//
```
Comment (ZappoMan) [inline libraries/fbx/src/FBXReader.h]: what happened to this file???

File: libraries/render-utils/src/Skinning.slh [hunk 3]
```diff
@@ -1,70 +1,70 @@
-<!
```
Comment (ZappoMan) [inline libraries/render-utils/src/Skinning.slh]: what happened to this file???

**Comment by ZappoMan** [comment]
this PR looks messed up... it has a bunch of files you probably shouldn't be changing in it

**Comment by ericrius1** [comment]
I'm not sure how this happened... will close and reopen.. It's not ready yet anyway, it turns out

**Pull request closed without merging**